metapi/metapi.js

```diff
@@ -129,9 +146,12 @@ window.metapi = {};
     })
     .done(function (data) {
       var items = data.items;
-      if (items.length > 0 && items[0]) {
-        metapi.postCache.add(ident, items[0]);
-        callback(new metapi.Response(true, items[0]));
+      if (items.length > 0) {
+        for (var k = 0; k < items.length; k++) {
+          metapi.postCache.add(items[k].link, items[k], {overwrite: true}); // Overwrite: "urls to be loaded" aren't cached.
```

Comment:
Bear in mind that the `link` field may not be present in the response. It's probably better to cache it against the ident it was passed to the function with, somehow.